src/k8s/cmd/k8s/k8s_status.go

```diff
@@ -34,12 +39,13 @@ var (
 				return fmt.Errorf("failed to get cluster status: %w", err)
 			}
 
-			// TODO: make this nice and bright
-			fmt.Printf("Number of nodes in the cluster: %d\n", len(clusterStatus.Members))
-			fmt.Printf("HA cluster: %t\n", clusterStatus.HaClusterFormed())
-			fmt.Println()
-			for _, component := range clusterStatus.Components {
-				fmt.Printf("%s: %s\n", component.Name, component.Status)
+			f, err := formatter.New(statusCmdOpts.outputFormat, cmd.OutOrStdout())
+			if err != nil {
+				return fmt.Errorf("failed to create formatter: %w", err)
+			}
+			err = f.Write(clusterStatus)
+			if err != nil {
+				return fmt.Errorf("failed to write output: %w", err)
 			}
 
 			return nil
```

Comment:
i would probably make this into a `formatter.Print(any) error`, so that all commands can do

```
return formatter.Print(format, _something_)
```